proj/src/main/webapp/notifications.js

```diff
@@ -105,12 +105,14 @@ function sendNotificationForChat(message) {
     const tag = sessionStorage.activeChatTag;
     const chatId = sessionStorage.activeChatId;
     const name = message.senderDisplay;
-    notificationBody = name+' has sent a message:\n'+message.content
+    notificationBody = name+' has sent a message:\n\n'+message.content
     let payload = {
         "to": getTopic(tag,chatId),
         "notification": {
             "title": tag,
-            "body": notificationBody
+            "body": notificationBody,
+            "icon": '/static/images/friendsChatting.png',
+            "click_action": 'https://8080-9f73b32e-6ccb-426e-8d6e-c03d6f324666.us-west1.cloudshell.dev/static/chat.html'
```

Comment:
This url will open just the chat rather than opening a specific chat. Unfortunately we don't have something built in to redirect them to a specific chat yet, though , so this is fine.